### DISCUSSION TITLE
rcloneによるGoogle Driveへのデータ転送について

### DISCUSSION THREAD

**Discussion by Junna-Kawasaki** [nig-sc/nigsc_homepage2#461] [Q&A] [answered]
平素よりお世話になっております。
遺伝研スパコンのデータをGoogle Driveにバックアップしたいと考えているのですが、rcloneはインストールされていないようでした。
代わりの推奨コマンドがありましたら、ご教示いただけますと幸いです。
どうぞよろしくお願いいたします。

**Comment by oogasawa** [comment, marked as answer]
rcloneはルート権限がなくても自分のホームディレクトリにインストールできるはずなので以下の手順試してみていただけますでしょうか。

1. Rcloneのダウンロードと解凍

 ```
  mkdir -p ~/local/src
  cd ~/local/src
  curl -O https://downloads.rclone.org/rclone-current-linux-amd64.zip
  unzip rclone-current-linux-amd64.zip
```

2. 解凍したディレクトリ内の`rclone`実行ファイルを、パスが通ったディレクトリにコピーします。
 
```
   mkdir -p ~/local/bin
   export PATH=$HOME/local/bin:$PATH
   mv rclone-*-linux-amd64/rclone ~/local/bin/
```

3.  `rclone`コマンドを実行して動作を確認します。
```
   rclone config
```
